content/docs/details/roadie-local/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Roadie Local
+publishedDate: '2025-05-29T16:00:00.0Z'
+description: How to run Roadie locally
+---
+
+## Prerequisites
+
+1. You must have the `roadie-local` CLI installed.
+2. Ensure your docker compose version is at least `2.23.1`. If not, please update to the latest version [using the official docs](https://docs.docker.com/compose/).
+
+
+## Getting Started
+
+
+1. Set the the license environment variable
+
+```bash
+export ROADIE_LICENSE=<license_key>
+```
+
+2. Run Roadie Local
+
+The following command will start Roadie on a URL. Open the URL in your browser.
+
+```bash
+./roadie-local start
+```
+
+3. Log in
+
+The following users are hardcoded to get you started.
+
+| Username | Password |
+|-----------|------------|
+| admin | roadie |
+| maintainer | roadie |
+| reader | roadie |
+
+
+4. To stop Roadie Local
+
+```bash
+./roadie-local stop
+```
+
+## More command line options
+
+### Command Line Options
+
+The CLI supports additional command line arguments to configure your environment:
+
+```bash
+# Basic usage - provide your OAuth config file
+./roadie-local start -c /path/to/your/config.yaml
```

Comment:
Best to just refer to the special docs for this.. its not going to make sense in this page is it?